srcpkgs/aerc/template

```diff
@@ -1,29 +1,33 @@
 # Template file for 'aerc'
 pkgname=aerc
-version=0.7.1
+version=0.9.0
 revision=1
 build_style=go
-hostmakedepends="scdoc git"
+go_import_path="git.sr.ht/~rjarry/aerc"
+go_build_tags=notmuch
+go_ldflags="-X main.Version=${version}"
+hostmakedepends="scdoc"
 makedepends="libnotmuch-devel"
 short_desc="Terminal email client"
 maintainer="shtayerc <david.murko@mailbox.org>"
 license="MIT"
 homepage="https://aerc-mail.org"
 distfiles="https://git.sr.ht/~rjarry/aerc/archive/${version}.tar.gz"
-checksum=e149236623c103c8526b1f872b4e630e67f15be98ac604c0ea0186054dbef0cc
+checksum=b5901feb37a55edd1f713e76c1012ac3fc0757202ddacd7d388cc7ce60638023
 
-do_configure() {
-	:
-}
+post_install() {
+	make doc
+	for f in *.{1,5,7}; do
+		vman $f
+	done
 
-do_build() {
-	make ${makejobs} PREFIX=/usr GOFLAGS=-tags=notmuch
-}
+	vmkdir usr/share/aerc
+	vcopy config/*.conf usr/share/aerc
+	vcopy filters usr/share/aerc
+	vcopy templates usr/share/aerc
+	vinstall config/default_styleset 644 usr/share/aerc/stylesets default
 
-do_install() {
-	make STRIP=true PREFIX=/usr DESTDIR=${DESTDIR} install
-}
+	vinstall contrib/aerc.desktop 644 usr/share/applications
```

Comment:
This is not really maintainable. Would something like this work? `do_install() { make install }`

Reply:
We would have to patch Makefile then. Just running `make install` starts building aerc again.

Reply:
@paper42 Done.

Reply:
Hmm, that's unfortunate, do you think we could convince upstream to have a install-data make target? Then we could drop the patch and just set `make_install_target="install-data"`

Reply:
I think not. We are not compiling in the recommended way (using make).